src/Commands/CommandFactory.cs

```diff
@@ -66,17 +66,37 @@ public CommandFactory(IServiceProvider serviceProvider, ILogger<CommandFactory>
 
     public IReadOnlyDictionary<string, IBaseCommand> AllCommands => _commandMap;
 
-    public IReadOnlyDictionary<string, IBaseCommand> GroupCommands(string groupName)
+    public IReadOnlyDictionary<string, IBaseCommand> GroupCommands(string commaSeparatedGroupNames)
     {
-        foreach (CommandGroup group in _rootGroup.SubGroup)
+        var groupNames = commaSeparatedGroupNames.Split(',', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
```

Comment:
I want to make sure that --service isn't required.  It can still be blank and start all right?